app/Models/Foundation/Summit/Summit.php

```diff
@@ -431,6 +431,12 @@ class Summit extends SilverstripeBaseModel
      */
     private $allow_update_attendee_extra_questions;
 
+    /**
+     * @ORM\Column(name="AllowAutomaticReminderEmails", type="boolean")
```

Comment:
please rename db field to **RegistrationAllowAutomaticReminderEmails**

Reply:
Done